doc/md/aggregate.md

```diff
@@ -84,4 +84,57 @@ func Do(ctx context.Context, client *ent.Client) {
 		}).
 		Scan(ctx, &users)
 }
+```
+
+## Having + Group By
+
+This example requires custom SQL modifiers. See how to enable them [here](features.md#custom-sql-modifiers)
```

Comment:
Please change URL to https://entgo.io/docs/feature-flags/#custom-sql-modifiers

Reply:
fixed